dev-requirements.txt

```diff
@@ -0,0 +1,4 @@
+flask==1.0.2
```

Comment:
Why do we need Flask as a dev requirement?

Reply:
removed